generate.js

```diff
@@ -98,19 +101,20 @@ function generate(component, options) {
     template(options.isFunctional ? 'stateless.js' : 'stateful.js'),
   ]
   const styleFiles = [template('styles.css')]
+  const cssExtension = options.cssExtension.replace(/^\./, '')
 
   scriptFiles.forEach(script => {
     const { name } = path.parse(script.filePath)
     const { content } = implementTypeChecking(options.typeCheck, script.content)
     const filePath = path.join(componentPath,
       `${name === 'index' ? 'index' : fileName}.js`
     )
-    save(componentName, fileName, filePath, content)
+    save(componentName, fileName, filePath, cssExtension, content)
```

Comment:
Just a note to myself. This should probably not be coupled with `script` saving. Should create a separate transform function to handle this stuff.